test/Microsoft.Azure.Devices.Edge.Test.Common/linux/PackageManagement.cs

```diff
@@ -60,10 +60,27 @@ public string[] GetInstallCommandsFromLocal(string path)
             };
         }
 
-        public string[] GetInstallCommandsFromMicrosoftProd(Option<Uri> proxy)
+        public string[] GetInstallCommandsFromMicrosoftProd(Option<Uri> proxy) => this.packageExtension switch
         {
-            throw new NotImplementedException("aziot-edge and aziot-identity-service currently aren't available in package repos");
-        }
+            SupportedPackageExtension.Deb => new[]
+            {
+                $"curl https://packages.microsoft.com/config/ubuntu/18.04/multiarch/prod.list > /etc/apt/sources.list.d/microsoft-prod.list",
```

Comment:
I see this is currently a draft, apologies if I'm too early...could we use `{this.os}` and `{this.version}` here too, so that we can run the end-to-end tests on any supported platform and it will "just work"?

Reply:
Right, @damonbarry, this will now work on RPi. I'm going to change it back to dynamic os/version. The problem is that often devs use Ubuntu 20.04 which does not work. Probably I need to put some special casing for 20.04 to use 18.04 repo. Any thoughts?

Reply:
Oh, why does 20.04 not work?

Reply:
@damonbarry, There is no `aziot-edge` package in `https://packages.microsoft.com/config/ubuntu/20.04/prod.list`. My knowledge is limited here... I guess we just don't publish there?

Reply:
Ah, I see. Yes 20.04 packages are coming but aren't there quite yet.

Reply:
Could we use dynamic if values present and default to general statement otherwise? That way we are future-proof?

Reply:
Thanks for your input folks. This is the new piece of logic that I come up with, wince we really support only two options now (18.04 and raspbian stretch).